src/pages/kimseulki/Main/Main.js

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from "react";
+import "./Main.scss";
+import "../../../style/reset.scss";
+import { Link } from 'react-router-dom';
+import { useNavigate } from 'react-router-dom'; 
+
+const Main = () => {
+  const [color, setColor] = useState('red');
+
+  const navigate = useNavigate();
+
+  const goToLogin = () => {                                    
+    navigate('/kim-login');
+  };
+
+  return (
+    <div className="main">
+      <h1>메인페이지</h1>
+      <nav>
+        <ul>
+          <li>
+            <Link to='#!'><img className="main_icon_logo" src="images/김슬기/main_icon_logo.png"/></Link>
```

Comment:
전체적으로 className이 `snake_case`로  작성되어있습니다. 기본적으로 JS에서는 `camelCase`를 사용하길 권장하고있기때문에 `snake_case`를 사용하는 모든 부분을 `camelCase`로 수정해주세요
```suggestion
            <Link to='#!'><img className="mainIconLogo" src="images/김슬기/main_icon_logo.png"/></Link>
```